### DISCUSSION TITLE
Allow custom `biome.json` route

### DISCUSSION THREAD

**Discussion by BoscoDomingo** [biomejs/biome#868] [Ideas]
I would like to be able to load a `biome.json` that isn't on the root of the workspace (instead in `./config` as it is shared by multiple projects within the same repo)

**Comment by nhedger** [comment]
Hey,

Love the idea, but the VS Code extension is currently not responsible for loading the configuration file.

This is done by the LSP server, which expects the file to be at the root of the workspace.

I'll move this discussion over so it's in the right place.

**Comment by MrTact** [comment]
You can fake this right now by passing `--config-path`, if you are using something like an npm script or a shell script.

Edit: sorry, I missed that this is for integration with VS Code, so this trick probably isn't going to help you.
